### DISCUSSION TITLE
Cannot download required documents in the Welcome interface

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#651] [Q&A]
The software has no download speed when I check the system monitor.
I'm in China.

**Comment by mirkobrombin** [comment]
Could it be that some IPs have been blocked?

**Reply by ghost** [reply]
Should be so, but I'm not quiet sure. I dont know whether others have the same problem. And it still didnt work when I used a VPS in global proxy.

**Reply by mirkobrombin** [reply]
> Should be so, but I'm not quiet sure. I dont know whether others have the same problem. And it still didnt work when I used a VPS in global proxy.

There was a similar issue but I can't find it anymore :/

**Reply by ghost** [reply]
Can we download the required files directly and deploy them ourselves?

**Reply by mirkobrombin** [reply]
> Can we download the required files directly and deploy them ourselves?

sure, you can find all the downloads in the components repository of this GitHub

**Reply by ghost** [reply]
Thank you！
